src/components/forms/OnboardingProfessional.tsx

```diff
@@ -92,6 +97,7 @@ export default function OnboardingProfessional() {
             </div>
           </div>
         </div>
+        <button type="submit"> TESTE ENVIAR </button>
       </Form>
       <pre>
         <code>{JSON.stringify(values, null, 2)}</code>
```

Comment:
I believe this code could be passed to a const
`JSON.stringify(values, null, 2)`